src/plugins/auto/CMakeLists.txt

```diff
@@ -14,14 +14,12 @@ if(ENABLE_AUTO AND ENABLE_MULTI)
 
     ie_add_plugin(NAME ${TARGET_NAME}
                   DEVICE_NAME "AUTO"
-                  PSEUDO_PLUGIN_FOR "MULTI"
-                  DEFAULT_CONFIG "MULTI_WORK_MODE_AS_AUTO:YES")
+                  PSEUDO_PLUGIN_FOR "MULTI")
 elseif(ENABLE_AUTO)
     ie_add_plugin(NAME ${TARGET_NAME}
                   DEVICE_NAME "AUTO"
                   SOURCES ${SOURCES}
-                  VERSION_DEFINES_FOR plugin.cpp
-                  DEFAULT_CONFIG "MULTI_WORK_MODE_AS_AUTO:YES")
+                  VERSION_DEFINES_FOR plugin.cpp)
```

Comment:
Do we have full test on local CI?

Reply:
Not yet, let me trigger the test on Local CI now.